README.md

```diff
@@ -68,7 +78,7 @@ Select all that apply:
 ```
 [] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
+[x] `$(".post").html()`
 [] `document.querySelectorAll(".post")[0].innerHTML`
 [] `document.querySelectorAll(".post").innerHTML`
 ```
```

Comment:
Check the [solution](https://github.com/ga-wdi-quizzes/week-04/tree/solution#jquery).